src/explore-education-statistics-frontend/server.js

```diff
@@ -79,20 +73,16 @@ async function startServer(port = process.env.PORT || 3000) {
             process.env.CONTENT_API_BASE_URL.replace('/api', ''),
             'data:',
             'https://www.google-analytics.com/',
-            'https://insights.hotjar.com',
-            'https://static.hotjar.com',
-            'https://script.hotjar.com',
-          ],
-          fontSrc: [
-            "'self'",
-            'https://static.hotjar.com',
-            'https://script.hotjar.com',
           ],
+          fontSrc: ["'self'"],
           connectSrc:
             process.env.NODE_ENV !== 'production' ? ['*'] : cspConnectSrc,
-          frameSrc: ["'self'", 'https://vars.hotjar.com '],
+          frameSrc: [
+            "'self'",
+            'https://department-for-education.shinyapps.io/',
+          ],
           frameAncestors: ["'self'"],
-          childSrc: ["'self'", 'https://vars.hotjar.com'],
```

Comment:
Not necessary for this PR but since we're getting rid of the hotjar CSP, maybe it'd be worth deleting the hotjar import from `_app.tsx` since it's unused?